.expeditor/scripts/shared.sh

```diff
@@ -123,9 +123,8 @@ s3_channel_url_root() {
 }
 
 s3_sync() {
-	local src="${1}"
-	local dst="${2}"
-	aws s3 sync "${src}" "${dst}"
+    # ex. s3_sync "src" "dst" "option1" "option2 <value>" ...
+    aws s3 sync "$@"
```

Comment:
:thinking: It appears that the website deploy is the only consumer of `s3_sync`. As such, I don't really see that this abstraction really buys us anything. I think you could just remove this function and call `aws s3 sync . "s3://$AWS_BUCKET" --delete` directly in `website-deploy.sh` and call it a day.

Reply:
Handled in https://github.com/habitat-sh/habitat/pull/7829/commits/316dbb662531496c918c61c95a7e8c40403bf2bc